.circleci/config.yml

```diff
@@ -124,6 +159,15 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+      - build_all_locales:
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
I think this will run build and then build_all_locales for all tagged builds, we can probably just remove the regular build

Reply:
Done